cmd/envtool/envtool.go

```diff
@@ -225,9 +226,77 @@ func runCommand(command string, args []string, stdout io.Writer, logger *zap.Sug
 	cmd := exec.Command(bin, args...)
 	logger.Debugf("Running %s", strings.Join(cmd.Args, " "))
 
-	cmd.Stdout = stdout
 	cmd.Stderr = os.Stderr
 
+	if args[1] == "-json" {
```

Comment:
I think we can remove that `if` and always add `-json` flag when tests are run through envtool